packages/notification-center/src/components/notification-center/components/layout/Layout.tsx

```diff
@@ -9,26 +9,33 @@ import { INovuTheme } from '../../../../store/novu-theme.context';
 import { UserPreference } from '../user-preference/UserPreference';
 import { UserPreferenceHeader } from './header/UserPreferenceHeader';
 
+export enum Screens {
```

Comment:
```suggestion
export enum ScreensEnum {
```

Maybe worth adding an enum ending, since we use it in other places aswell.